manifests/h/Hashicorp/Terraform/Beta/1.6.0-beta1/Hashicorp.Terraform.Beta.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.2.10 $debug=NVS0.CRLF.7-3-6.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Hashicorp.Terraform.Beta
+PackageVersion: 1.6.0-beta1
+InstallerLocale: en-US
+InstallerType: zip
+FileExtensions:
+- tf
+ReleaseDate: 2023-08-31
+Installers:
+- Architecture: x86
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: ./terraform.exe
+  - RelativeFilePath: a
```

Comment:
```suggestion
```
Hi @kdpuvvadi,

I think this extra entry might be causing the validation error.